en/lessons/basics/documentation.md

```diff
@@ -145,6 +145,7 @@ First let's create a Mix project for our application:
 $ mix new greet_everyone
 
 * creating README.md
+* creating .formatter.exs
```

Comment:
When we change files we need to bump the version at the top otherwise translator won't be notified.

Reply:
That previous pull request is already accepted, so I cannot do it :) And I also reading and studying elixir step-by-step, so I cannot predict what else is missing

Reply:
If you could bump this file to `1.0.3` I can address the other languages.

Reply:
For your own edification, we use these versions to generate a translation report (developed by @gemantzu): https://elixirschool.com/ru/report/

You can only see the report if you're on a translated version of the site.

Reply:
Yes, tnx, I got the point. I saw this message in Russian